packages/client/components/SpotlightModal.tsx

```diff
@@ -168,9 +208,11 @@ const SpotlightModal = (props: Props) => {
             />
           </SearchItem>
         </SelectedReflectionSection>
-        <SimilarReflectionGroups>
-          {reflectionGroupsCount === 0 ? <SpotlightEmptyState /> : null}
-        </SimilarReflectionGroups>
+        <SimilarGroups>
+          <Suspense fallback={''}>
```

Comment:
Suspense is required here as the query in `SpotlightGroup` can be loading. If we remove this, remotely group & ungroup results in the Spotlight, notice that the modal flashes